### ISSUE TITLE
can we have a less file for each icon so we can include only the icons we want to use

### ISSUE THREAD

**Issue by dowda** [WestpacCXTeam/GUI-source#168]
Is this possible.  We are running into the problem of including 275KB of less just including symbols.data.svg.css

I'd like to know if it's possible to get an individual less file per icon so I can then build out what I want into symbols.data.svg.css

cheers
jake the muss

**Comment by dominikwilkowski** [comment]
Hey Dowie

Yeah that's the very reason for why we created separate icons modules. For us providing a module per icon means we blow up the blender to more than 150 modules (for currently 47) which didn't seem sensible at the time. We then decided to group them in meaningful selections with not more than 13 icons in each.
Happy to talk it through though if you have ideas.

Just to clarify: We don't actually have `less` code for the icons or any svgs. We only have svg files that are run through some processes that generate the three css files and the png fallbacks. The automation works quite nice and has enabled us to maintain more than 500 icons (for all brands).
